src/le.py

```diff
@@ -2035,6 +2069,8 @@ def __init__(self):
         # Enabled fine-grained logging
         self.debug = False
         # All recognized events are logged
+        self.debug_cmds = False
```

Comment:
Probably shouldn't refer to commands as "events" in the comment. Best to stick with one name for each thing